src/main/java/org/walkmod/javalang/compiler/types/TypesLoaderVisitor.java

```diff
@@ -159,7 +164,7 @@ private String getSymbolName(String name, boolean imported) {
                simpleName = aux;
             }
          }
-         simpleName = simpleName.replaceAll("\\$", ".");
+         simpleName = simpleName.replace("$", ".");
```

Comment:
Yes ....?  
non regexp replace() always replaces all. 
No functional change here.
All ocurrences of '$' are replaces with '.'
I should have used the character version instead of the string version though.